### ISSUE TITLE
Configuration editor won't resize after being made screen-filling

### ISSUE THREAD

**Issue by Lucwousin** [DarthAffe/KeyboardAudioVisualizer#7]
Trying to make the editor smaller after you make it fullscreen (dragging it to the top of your monitor), nothing will happen (minimising to tray will still work). This is happening on both my monitors.

Setup: Windows 10 pro 64 bit, 2 monitors.

**Comment by DarthAffe** [comment]
Yeah, that is an issue with aero-snap. The window isn't intended to be fullscreened but this is a way to get it into that state. Right now the only way to get it out of it is by using Win+ArrowDown (or left/right).
I'll check if I can prevent that from happening.

**Comment by DarthAffe** [comment]
Well the only real reason for that is the fact that I didn't want to invest time in investigating a weird behavior decorationless windows have when maximized. It causes them to be bigger then the monitor they are maximized on (at least on win 7 - I've no idea if other version are affected by this too - the window is partially visible on the monitors left and right too). 

I'll take a look at it again (but I expect the next version to take a while).
